bin/compile_assets.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Script to generate assets using `yarn encore`.
+# It checks SYMFONY_ENV to ensure assets are generated for correct production/development environment.
+
+if [ "${SYMFONY_ENV}" == "dev" ] ; then
+    yarn encore dev
+else
+    yarn encore prod
```

Comment:
[AFAICS](https://symfony.com/doc/current/frontend/encore/simple-example.html#configuring-encore-webpack):
```suggestion
    yarn encore production
```

Reply:
No, `prod` is supported now.

Reply:
i was going to say same thing as @alongosz but if prod is supported, this looks good to me. +1

Reply:
Ref: https://github.com/symfony/webpack-encore/blob/v0.26.0/bin/encore.js#L67

Reply:
Yep it's supported now. I used `prod` because it reflects default environment in Symfony which might be more familiar for developers. But I have no strong opinion on that.

Reply:
Ok, so Symfony doc is outdated.